### PR TITLE
use near-abi-rs

### PR DESCRIPTION
**Pull request by miraclx**
https://github.com/near/near-abi-rs/pull/1
https://github.com/near/near-sdk-rs/pull/889

### CODE REVIEW

**Review by austinabell — APPROVED** [review]

**Comment by miraclx** [comment]
One of our CI runs seems to have gotten stuck following an issue with GitHub Actions.

https://www.githubstatus.com/incidents/pmw828sz2w7k

https://github.com/orgs/community/discussions/30079#discussioncomment-3472190

File: cargo-near/Cargo.toml
Comment (austinabell) [inline cargo-near/Cargo.toml]: Can we use `0.1.0-pre.0`? https://crates.io/crates/near-abi/0.1.0-pre.0

Comment (miraclx) [reply cargo-near/Cargo.toml]: has already been updated.

Comment (austinabell) [reply cargo-near/Cargo.toml]: oh, oops, must have been looking at an outdated commit

**Review by austinabell — APPROVED** [review]

**Review by itegulov — APPROVED** [review]

File: integration-tests/templates/_Cargo.toml
Comment (miraclx) [inline integration-tests/templates/_Cargo.toml:17]: should probably wait for an sdk prerelease before merging this in, or wdyt?

Comment (itegulov) [reply integration-tests/templates/_Cargo.toml:17]: What is missing for us to release 4.1.0-pre.2? Are we waiting on https://github.com/near/near-sdk-rs/pull/891 and https://github.com/near/near-sdk-rs/pull/893 to be pulled in?

Comment (miraclx) [reply integration-tests/templates/_Cargo.toml:17]: We can cut the prerelease without those two. Since we still need to properly review the ABI embedding.

Comment (itegulov) [reply integration-tests/templates/_Cargo.toml:17]: Ok, I suggest we ask @austinabell to cut 4.1.0-pre.2 and wait for the release then

Comment (austinabell) [reply integration-tests/templates/_Cargo.toml:17]: ah, I'll do that now

Comment (austinabell) [reply integration-tests/templates/_Cargo.toml:17]: done

**Review by austinabell — APPROVED** [review]
updated versions lgtm

**Pull request merged**
